src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -360,22 +377,18 @@ def shim_to_feedback(
                 feedback_event["contexts"]["trace"] = event.data["contexts"]["trace"]
 
             feedback_event["timestamp"] = event.datetime.timestamp()
-            feedback_event["level"] = event.data["level"]
             feedback_event["platform"] = event.platform
             feedback_event["level"] = event.data["level"]
             feedback_event["environment"] = event.get_environment().name
             feedback_event["tags"] = [list(item) for item in event.tags]
 
-        else:
```

Comment:
Wait why are we deleting this else condition? This relates to the platform and level lines below too

Reply:
oops that was unintentional. GOOD CATCH